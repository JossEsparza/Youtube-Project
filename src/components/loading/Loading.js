import React from 'react';
import './Loading.scss';

const Loading = () => {
  return (
    <div className="load-wrapp">
      <div className="load-5">
        <div className="ring-2">
          <div className="ball-holder">
            <div className="ball" />
          </div>
        </div>
      </div>
    </div>
  );
};

export default Loading;
