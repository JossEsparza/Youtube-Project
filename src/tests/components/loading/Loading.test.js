import { shallow } from 'enzyme';
import Loading from '../../../components/loading/Loading';

describe('Tests the Loading', () => {
  test('should be displayed correctly', () => {
    const wrapper = shallow(<Loading />);
    expect(wrapper).toMatchSnapshot();
  });
});
