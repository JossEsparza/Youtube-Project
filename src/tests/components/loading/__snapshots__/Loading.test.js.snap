// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests the Loading should be displayed correctly 1`] = `ShallowWrapper {}`;
